dotnet/targets/Xamarin.Shared.Sdk.targets

```diff
@@ -573,6 +573,9 @@
 			<!-- Set default ValidateObjectPointers value -->
 			<_ValidateObjectPointers Condition="'$(_ValidateObjectPointers)' == ''">false</_ValidateObjectPointers>
 
+			<!-- Set default DisposeTaggedPointers value -->
+			<_DisposeTaggedPointers Condition="'$(_DisposeTaggedPointers)' == ''">true</_DisposeTaggedPointers>
```

Comment:
Isn't the default supposed to be `false`? How does the test pass? (Or, am I just misreading it?)

Reply:
Yes, this was just a quick test run on CI to see if doing it this way would break any test, the PR isn't complete at all.